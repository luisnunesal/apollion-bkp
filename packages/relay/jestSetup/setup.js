const { JSDOM } = require('jsdom');
const server = require('./testServer');

module.exports = async () => {
  const httpServer = await server.listen(5000, () => {
    console.log(`\n test server running at: \n${JSON.stringify(httpServer.address(), null, 2)}`);
  });

  Object.assign(global, { httpServer });
};
