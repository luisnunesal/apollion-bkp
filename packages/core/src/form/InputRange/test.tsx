import React from 'react';

import { InputRange } from '.';

import { render, screen } from '../../__tests__/setup';

describe('Default Input Range valid props', () => {
  it('Snapshot should match default', () => {
    render(<InputRange min={0} max={200} value={50} onChange={(value: number) => null} />);
    expect(screen).toMatchSnapshot();
  });
});
