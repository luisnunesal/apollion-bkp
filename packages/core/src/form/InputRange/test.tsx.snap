// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default Input Range valid props Snapshot should match default 1`] = `
<div
  class="sc-AxirZ hGDxvN"
>
  <div
    class="sc-AxirZ VZfRX"
    height="100%"
    width="100%"
  >
    <div
      class="sc-fznJRM lgfvyv"
    >
      <span
        class="sc-AxmLO NrfFd"
        color="primary"
      >
        50
      </span>
    </div>
    <div
      class="sc-AxirZ sc-fzoLag izmZQM"
      flexDirection="row"
      width="100%"
    >
      <input
        class="sc-fzoyAV fFyzSZ sc-fzoXzr bgOjVx"
        max="100"
        min="0"
        step="25"
        type="range"
        value="50"
      />
    </div>
  </div>
</div>
`;
